src/docs/devices/Apollo-Automation-AIR-1/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Apollo Air Quality Sensor (AIR-1)
+date-published: 2023-11-11
+type: sensor
+standard: global
+board: esp32
+project-url: https://github.com/ApolloAutomation/AIR-1/blob/main/Integrations/ESPHome/AIR-1.yaml
+difficulty: 1
+---
+
+The Apollo Automation AIR-1 is an air quality sensor with the following features:
+
+- SEN55 PM1, PM2.5, PM4, PM10, VOC, NOx, Temperature and Humidity
```

Comment:
For the sensor list, I think it'd be cool to link to the ESPHome documentation for that specific sensor. Another option would be an official product page or datasheet, but I think the ESPHome implementation details for that sensor would be more relevant.

Just a thought, though!